runtime/vm/ValueTypeHelpers.cpp

```diff
@@ -250,4 +250,15 @@ areValueTypesEnabled(J9JavaVM *vm)
 #endif /* defined(J9VM_OPT_VALHALLA_VALUE_TYPES) */
 }
 
+j9object_t
+getDefaultValue(J9Class* clz)
+{
+	Assert_VM_notNull(clz);
+	Assert_VM_true(clz->initializeStatus == J9ClassInitSucceeded);
+	Assert_VM_notNull(clz->flattenedClassCache);
```

Comment:
This assertion can also be removed as you are using `clz->flattenedClassCache->defaultValue` the next line.

Reply:
ah ok, I thought it might be preferable to a segfault